client/package.json

```
@@ -355,9 +355,9 @@
 					"default": {
 						"v": "674a514867b1",
 						"viperToolsPath": {
-							"windows": "%APPDATA%\\Viper\\",
-							"linux": "$HOME/.config/Viper",
-							"mac": "$HOME/Library/Application Support/Viper"
+							"windows": "%APPDATA%\\Roaming\\Code\\User\\globalStorage\\viper-admin.viper\\Local",
+							"linux": "$HOME/.config/Code/User/globalStorage/viper-admin.viper/Local",
+							"mac": "$HOME/Library/Application Support/Code/User/globalStorage/viper-admin.viper/Local"
```

```suggestion
							"windows": "%APPDATA%\\Roaming\\Code\\User\\globalStorage\\viper-admin.viper\\Local\\ViperTools",
							"linux": "$HOME/.config/Code/User/globalStorage/viper-admin.viper/Local/ViperTools",
							"mac": "$HOME/Library/Application Support/Code/User/globalStorage/viper-admin.viper/Local/ViperTools"
```

This would be in annnalogy to the download locations for `Stable` and `Nightly`, and for creating a `Local` path on would only rename the path component.
For reference, I have
```
/Users/ggreif/Library/Application Support/VSCodium/User/globalStorage/viper-admin.viper/Nightly/ViperTools
```
@ArquintL what do you think?

---

Good catch. Thanks I prefer it this way too